### PR TITLE
Remove tshirt code

### LINKED ISSUE #4152: Remove t-shirt code
[there's quite a bit](https://github.com/DoSomething/dosomething/search?utf8=%E2%9C%93&q=shirt&type=Code)

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?

remove t-shirt option in shipment form and rewards
#### How should this be reviewed?

👀 
#### Any background context you want to provide?

There are still some references to `shirts` in the POT files. 
#### Relevant tickets

Fixes #4152 

cc @mikefantini

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc [hunk 1]
```diff
@@ -10,9 +10,6 @@ function paraneue_dosomething_form_alter_base(&$form, &$form_state, $form_id) {
   if(isset($form['actions']['submit'])) {
     $form['actions']['submit']['#attributes']['class'][] = 'button';
   }
-
-  // Check if this form needs Shirt Styles applied.
-  paraneue_dosomething_form_alter_shirt_options($form, $form_state, $form_id);
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc:15]: Curious if any corresponding CSS 🎨 we'd want to cleanup too? I recall Tom working on the fender side of this form.

**Comment by aaronschachter** [comment]
👀 👍

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc:15]: Yep, that's a [pattern in Forge](http://forge.dosomething.org/#forms-media-option-field)! Definitely something that could be removed next time we make a big ol' version bump.

Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc:15]: good call! created an issue in forge https://github.com/DoSomething/forge/issues/554

**Pull request merged**